### DISCUSSION TITLE
Build production branch twice with different env variables

### DISCUSSION THREAD

**Discussion by darkowic** [vercel/community#155] [Show & Tell]
Hi! Is it possible to build the `main` production branch with 2 different sets of env variables and deploy it to different domains?

I want to integrate with headless CMS Storyblok and I develop the page using Gatsby. I need 2 builds:
* preview.domain.com - with preview API key injected and code being able to use the Storyblok API
* domain.com - the production version with production API key

So, simply: `main` branch, 2 builds with different sets of env variables.

Thanks for help!

**Comment by Pieparker** [comment]
The easiest solution would be to use two projects; keep it all autonomous. From the top of my head, any other solution would involve an element of duplication (such as duplicating content across branches). Interested to hear what others would do!
